source/reference/ulimit.txt

```diff
@@ -104,6 +104,12 @@ system limits, as in the following example:
    -r: max rt priority            65
    -N 15:                         unlimited
 
+.. note::
+
+  If :binary:`~bin.mongod` or :binary:`~bin.mongos` processes are launched
+  using a systemwide process manager such as Upstart or systemd, they may
+  be subject to different resource limits.
```

Comment:
I think this note would be more helpful if it pointed out that ulimit considerations are already handled in MDB init scripts. See the init.d, service, upstart, suse files here:

https://github.com/mongodb/mongo/tree/master/debian
https://github.com/mongodb/mongo/tree/master/rpm

Even brew sets appropriate ulimit values here:

https://github.com/mongodb/homebrew-brew/blob/master/Formula/mongodb-community.rb

So instead of differentiating based on running shell vs system-started shell,  I would differentiate via user action required (TGZ install) vs no action required (package manager install).